deps/build.jl

```diff
@@ -40,6 +40,15 @@ end
 info("Found Jupyter version $jupyter_vers: $jupyter")
 
 #######################################################################
+# Get the latest syntax highlighter file.
+if isconda
+    highlighter = joinpath(Conda.LIBDIR, "python2.7", "site-packages", "notebook", "static",
+                           "components", "codemirror", "mode", "julia", "julia.js")
+    highlighter_url = "https://raw.githubusercontent.com/codemirror/CodeMirror/master/mode/julia/julia.js"
```

Comment:
If the stability of master is a concern, we could just use a specific commit and bump the commit we use here every now and then.